physics/CONV/progsigma_calc.f90

```diff
@@ -168,13 +174,13 @@ subroutine progsigma_calc (im,km,flag_init,flag_restart,flag_shallow,&
       enddo
 
       !sigmab
-      if(flag_init .and. .not. flag_restart)then
-         do i = 1,im
-            if(cnvflg(i))then
-               sigmab(i)=0.03
-            endif
-         enddo
-      else
+!      if(flag_init .and. .not. flag_restart)then
+!         do i = 1,im
+!            if(cnvflg(i))then
+!               sigmab(i)=0.03
+!            endif
+!         enddo
+!      else
          do i = 1,im
```

Comment:
I think you can remove the commented lines (if you plan on merging this into the UFS?)